src/containers/Layout/Layout.js

```diff
@@ -7,12 +7,31 @@ import Sidebar from "../../components/Sidebar/Sidebar";
 import Jokes from "../Jokes/Jokes";
 
 class Layout extends Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      jokesList: [],
+    }
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    const target = event.target;
+    const name = target.name;
+
+    fetch( 'https://api.chucknorris.io/jokes/random?category=' + name )
```

Comment:
May be its a good idea to create a constant for the base url: `https://api.chucknorris.io/jokes/` so you dont need to put it as a string in several places also it will be a much safer to change it if API changes.